tests/v0_tests/test_parallel.py

```diff
@@ -19,7 +19,7 @@ def setUp(self) -> None:
         except MarqoApiError as s:
             pass
         
-        self.para_params = {'server_batch_size':10, 'processes':2}
+        self.para_params = {'server_batch_size':10, 'processes':2, 'device': "cpu"}
```

Comment:
is this just to make this test a little more stable (given issues with parallel indexing?)

Reply:
yes, exactly. i did it in `test_logging` as well, since MP with cuda has some weird issues.

Actually i'm still having a slight issue with `test_logging`. When I run those tests alone, they pass, but when I run them as part of the whole test suite, they make marqo break witha  `Broken Pipe Error`. Pretty sure this is some issue with MP.

Reply:
fixed. just reran it again. this may have been caused by running marqo-os locally alongside cuda DIND, meaning i had 2 instances of marqo-os running.